census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java

```diff
@@ -37,6 +39,20 @@ public final class ObservabilityCensusConstants {
   static final Aggregation AGGREGATION_WITH_BYTES_HISTOGRAM =
       RpcViewConstants.GRPC_CLIENT_SENT_BYTES_PER_RPC_VIEW.getAggregation();
 
+  public static final MeasureDouble API_LATENCY_PER_CALL =
```

Comment:
For line 36 can you add a TODO item to denote the code movement. I know it applies to the previous PR

Reply:
Done